platforms/hyperledger-fabric/configuration/chaincode-install-instantiate.yaml

```diff
@@ -52,16 +52,17 @@
       loop: "{{ network['channels'] }}"
       when: add_new_org == 'false' and '2.' in network.version    
         
-    # This task invokes chaincode after the chaincode installation for the new organization to 
-    # be added in existing network
+    # This task invokes chaincode after the chaincode has been commited in v2 and
+    # after chaincode installation for the new organization to be added in existing network
     - name: Invoke chaincode
       include_role:
         name: "create/chaincode/invoke"
       vars:
         participants: "{{ item.participants }}"
         docker_url: "{{ network.docker.url }}"
+        approvers: "{{ item.endorsers }}"
```

Comment:
Add a default as this `item.endorsers` is not used by fabric v1.4.x code and this should be optional in network.yaml